File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NumarParticipantiSedinteComisieGlReportModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class NumarParticipantiSedinteComisieGlReportModel {

	private Integer totalGeneral;
	private List<NumarParticipantiSedinteComisieGlReportRowModel> rows;

	public Integer getTotalGeneral() {
		return totalGeneral;
	}

	public void setTotalGeneral(Integer totalGeneral) {
		this.totalGeneral = totalGeneral;
	}

	public List<NumarParticipantiSedinteComisieGlReportRowModel> getRows() {
		return rows;
	}

	public void setRows(List<NumarParticipantiSedinteComisieGlReportRowModel> rows) {
		this.rows = rows;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/numar-participanti-sedinte-comisie-gl-report.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { NumarParticipantiSedinteComisieGlReportRowModel } from "./numar-participanti-sedinte-comisie-gl-report-row.model";

@JsonObject
export class NumarParticipantiSedinteComisieGlReportModel {
    	
	@JsonProperty("totalGeneral", Number)
	public totalGeneral: Number = null;
	
	@JsonProperty("rows", [NumarParticipantiSedinteComisieGlReportRowModel])
	public rows: NumarParticipantiSedinteComisieGlReportRowModel[] = [];
}